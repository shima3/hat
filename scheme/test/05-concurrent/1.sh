#!/bin/bash
$* -e main examples/concurrent.sch
