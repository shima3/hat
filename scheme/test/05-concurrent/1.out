10
a
10
b
9
a
8
a
9
b
7
a
6
a
8
b
5
a
4
a
7
b
3
a
2
a
6
b
1
a
0
5
b
4
b
3
b
2
b
1
b
0
